tests/datasource/fluent/test_config.py

```diff
@@ -819,7 +819,8 @@ def test_config_substitution_retains_original_value_on_save(
     from great_expectations import get_context
 
     context = get_context(
-        context_root_dir=file_dc_config_file_with_substitutions.parent
+        context_root_dir=file_dc_config_file_with_substitutions.parent,
+        ge_cloud_mode=False,
```

Comment:
I've set the cloud env variables for all marker stages so had to make sure these didn't get a cloud context.
Maybe I should just set the env variables on the cloud marker test? Or maybe I should do both. These tests should be robust to the env variables.

Reply:
Doing things like this let's me get rid of that darwin check in __init__.py. Though I still see tests dying because I've set the env variables in all marker tests so I might only set them in the cloud ones.

Reply:
I will move the fixture I introduced and use that instead of setting the `ge_cloud_mode` parameter.